CHANGELOG.md

```diff
@@ -24,6 +24,12 @@ _None_
   [David Jennes](https://github.com/djbe)
   [#28](https://github.com/SwiftGen/SwiftGenKit/issues/28)
   [#31](https://github.com/SwiftGen/SwiftGenKit/issues/31)
+* Update Stecil to 0.9.0
```

Comment:
Please add a `.` and 2 spaces at the end of the description. Also, join these 2 changes together.